crates/update/build.rs

```diff
@@ -0,0 +1,5 @@
+#![allow(clippy::disallowed_macros)]
+fn main() {
+    let target = std::env::var("TARGET").unwrap();
+    println!("cargo::rustc-env=BUILD_TARGET={target}");
+}
```

Comment:
I'm not very familiar with massaging the build process - what's this for?

Reply:
For some reason you can't access the compilation target without it setting it in a build script like this. This is for knowing what release artifact to download.